### PR TITLE
[Slim] Add PHPLint tool to check syntax

### PR DESCRIPTION
**Pull request by ybelenko**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `3.3.x`, `4.0.x`. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

`PHPLint` is a tool that can speed up linting of php files by running several lint processes at once.

Ref: [overtrue/phplint](https://github.com/overtrue/phplint)

```bash
$ composer run phplint
> phplint ./ --exclude=vendor
phplint 1.0.2 by overtrue and contributors.

No config file loaded.

.......................................     40 / 87 (45%)
........................................     80 / 87 (91%)
........

Time: 3 secs    Memory: 2.0 MiB Cache: No

OK! (Files: 87, Success: 87)
```

### Upgrade note
We have to increase required PHP version to 5.5.9, because it's minimum version for `PHPLint` tool.

cc @jebentier @dkarlovi @mandrean @jfastnacht @ackintosh

### CODE REVIEW

**Comment by wing328** [comment]
> We have to increase required PHP version to 5.5.9, because it's minimum version for PHPLint tool.

@ybelenko Given that it's a breaking change with fallback (as you've labeled), please file the PR against 3.3.x instead (you can update this PR by editing the subject line, and you will see a drop-down menu to change the target branch)

**Comment by ybelenko** [comment]
@wing328 
What if I've based it on current `master` branch and not `3.3.x`. Do I need to rebase it on `3.3.x` and check everything again?

**Comment by wing328** [comment]
We sync changes from master to 3.3.x and then from 3.3.x to 4.0.x from time to time so you shouldn't need to rework this PR.

Please give it a try and you can always change it (target branch) to master if you want to.

File: modules/openapi-generator/src/main/resources/php-slim-server/composer.mustache [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   "require": {
-    "php": ">=5.5",
+    "php": ">=5.5.9",
     "slim/slim": "3.*",
     "tuupola/slim-basic-auth": "^3.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.8"
+    "phpunit/phpunit": "^4.8",
+    "overtrue/phplint": "^1.0"
```
Comment (ybelenko) [inline modules/openapi-generator/src/main/resources/php-slim-server/composer.mustache]: Can't find this line in `master` branch after merge. Looks like mege conflicts has been resolved incorrectly.

File: samples/server/petstore-security-test/php-slim/composer.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
   "require": {
-    "php": ">=5.5",
+    "php": ">=5.5.9",
     "slim/slim": "3.*",
     "tuupola/slim-basic-auth": "^3.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.8"
+    "phpunit/phpunit": "^4.8",
+    "overtrue/phplint": "^1.0"
```
Comment (ybelenko) [inline samples/server/petstore-security-test/php-slim/composer.json]: Can't find this line too.

File: samples/server/petstore/php-slim/composer.json [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
   "require": {
-    "php": ">=5.5",
+    "php": ">=5.5.9",
     "slim/slim": "3.*",
     "tuupola/slim-basic-auth": "^3.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.8"
+    "phpunit/phpunit": "^4.8",
+    "overtrue/phplint": "^1.0"
```
Comment (ybelenko) [inline samples/server/petstore/php-slim/composer.json]: And this one.

File: modules/openapi-generator/src/main/resources/php-slim-server/composer.mustache (hunk 1 above)
Comment (wing328) [reply modules/openapi-generator/src/main/resources/php-slim-server/composer.mustache]: My bad. I'll add it back shortly.

Comment (wing328) [reply modules/openapi-generator/src/main/resources/php-slim-server/composer.mustache]: Added back via 5ad471cc6. My apologies for the mistake when resolving the merge conflicts.

**Comment by wing328** [comment]
@ybelenko thanks again for the PR, which is included in the v3.3.0 minor release: https://twitter.com/oas_generator/status/1046941449609068544

**Pull request merged**